src/Steps/Release/PlanRelease.php

```diff
@@ -155,7 +161,7 @@ protected function generateChildReleases(LibraryRelease $parent)
         $childModules = $parent->getLibrary()->getChildrenExclusive();
         foreach ($childModules as $childModule) {
             // For the given child module, guess the upgrade mechanism (upgrade or new tag)
-            if ($parent->getLibrary()->isChildUpgradeOnly($childModule->getName())) {
+            if (!$this->getIncludeUpgradeOnly() && $parent->getLibrary()->isChildUpgradeOnly($childModule->getName())) {
```

Comment:
```suggestion
            if ($childModule->isUpgradeOnly() || $parent->isUpgradeOnly()) {
```

Along with removing the conditional that's wrapping [Line 176](https://github.com/silverstripe/cow/pull/159/files#diff-4807555863ed3e3bcd6d9479644b1454R176), this _should_ retain the existing upgrade-only behaviour, while allowing child modules of an upgrade-only recipe to be included in the plan.

However, I have a feeling that the determined tags for child modules will still be incorrect - they should reflect the versions declared in the Composer config for the recipe, rather than computing the latest version (as there may have been further releases of the child module since the last recipe release).

Reply:
Tested this theory, and it looks like this actually behaves perfectly fine.